docs/maya/muscle.md

```diff
@@ -142,30 +160,31 @@ In order to better visualize deformer constraints and attributes in the Maya vie
  - **Slide on Segment Constraints**: For each vertex with a slide on segment weight greater than 0.0, a line will be drawn from the mesh vertex to the closest point to its respective segment.
 
 <figure markdown>
   ![AdnMuscle debug](images/muscle_debug.png)
-  <figcaption><b>Figure 6</b>: AdnMuscle debug features. From left to right: Muscle Fibers, Attachment Constraints and Slide On Segment Constraints.</figcaption>
+  <figcaption><b>Figure 9</b>: AdnMuscle debug features. From left to right: Muscle Fibers, Attachment To Transform Constraints, Slide On Segment Constraints, Attachment To Geometry Constraints and Slide On Geometry Constraints.</figcaption>
 </figure>
 
 ## Advanced
 
-### Attachments
+### Targets
 
-Once the AdnMuscle deformer is created, it is possible to add and remove new attachments to the system. 
+Once the AdnMuscle deformer is created, it is possible to add and remove new targets to the system. Targets that are joints, locators, etc. will be treated as transform targets. In the other hand, mesh nodes will be considered geometry targets.
 
-- **Add attachments**:  
-    1. Select the transform nodes (one or more) to be assigned as attachments to the AdnMuscle.
+- **Add targets**:
+    1. Select the transform or mesh nodes (one or more) to be assigned as targets to the AdnMuscle.
     2. Select the mesh that has the AdnMuscle deformer applied.
-    3. Press the ![Add Attachments button](images/adn_add_attachment.png){style="width:4%"} button in the AdonisFX shelf or press *Add Attachments* in the AdonisFX menu from the Edit Muscle submenu.
-- **Remove attachments**:
-    1. Select one or more transform nodes that are assigned as attachments to the muscle.
+    3. Press the ![Add Targets button](images/adn_add_target.png){style="width:4%"} button in the AdonisFX shelf or press *Add Targets* in the AdonisFX menu from the Edit Muscle submenu.
+- **Remove targets**:
+    1. Select one or more transform or mesh nodes that are assigned as targets to the AdnMuscle.
     2. Select the mesh that has the AdnMuscle deformer applied.
-    3. Press the ![Remove Attachments button](images/adn_remove_attachment.png){style="width:4%"} button in the AdonisFX shelf or press *Remove Attachments* in the AdonisFX menu from the Edit Muscle submenu. 
-    4. Alternatively, if only the mesh with the AdnMuscle deformer is selected, when pressing the ![Remove Attachments button](images/adn_remove_attachment.png){style="width:4%"} button, all attachments will be removed.
+    3. Press the ![Remove Targets button](images/adn_remove_target.png){style="width:4%"} button in the AdonisFX shelf or press *Remove Targets* in the AdonisFX menu from the Edit Muscle submenu. 
+    4. Alternatively, if only the mesh with the AdnMuscle deformer is selected, when pressing the ![Remove Targets button](images/adn_remove_target.png){style="width:4%"} button, all targets will be removed.
```

Comment:
```suggestion
    4. Alternatively, if only the mesh with the AdnMuscle deformer is selected, when pressing the ![Remove Targets button](images/adn_remove_target.png){style="width:4%"} button, all targets will be removed (transform and mesh targets).
```

Reply:
All these comments in muscle apply also to ribbon

Reply:
Done